CONTRIBUTING.md

```diff
@@ -72,6 +72,12 @@ PRs to resolve existing issues are fantastic. Two good places to start are the
 ["help wanted"](https://github.com/apostrophecms/apostrophe/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)
 and ["good first issue"](https://github.com/apostrophecms/apostrophe/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) issues.
 
+Please not that if you are using Windows for development, we strongly recommend
```

Comment:
We should put this in other places too, but it's an absolutely true statement that bears repeating to developers, so OK to have it here as well I think.